### ISSUE TITLE
Applying default control settings don't apply alternative keybinds

### ISSUE THREAD

**Issue by FreeSlave** [FWGS/xash3d-fwgs#1012]
E.g. default.cfg has
```
bind	UPARROW		+forward
bind	w			+forward
```
Xash3D seems to set the first bind only when "Use defaults" button is pressed.

**Comment by Vladislav4KZ** [comment]
By the way, when you enter in the console ``exec default.cfg``, the ``+forward`` command successfully binds to two buttons

**Comment by a1batross** [comment]
WTF is default.cfg?

**Comment by Vladislav4KZ** [comment]
> WTF is default.cfg?

This is the config that restores default keyboard settings.

**Comment by a1batross** [comment]
We don't use that anywhere. And it doesn't look like GoldSrc uses it, either.

For keyboard default settings, only `kb_def.lst` is used.

**Comment by Vladislav4KZ** [comment]
Anyway, the kb_def.lst has two buttons ``w`` and ``UPARROW`` that use the ``+forward`` command.

**Comment by iZarif** [comment]
> We don't use that anywhere. And it doesn't look like GoldSrc uses it, either.

Just to clarify, the steam version of Half-Life has a file default.cfg and the first time you run Xash3D key bindings are loaded from this file

**Comment by Vladislav4KZ** [comment]
> Just to clarify, the steam version of Half-Life has a file default.cfg and the first time you run Xash3D key bindings are loaded from this file

No, according to my observations, Goldsource does not use default.cfg to reset the keybindings, as well as Xash3D.

It uses the ``kb_def.lst`` file for restore the default keybindings.

**Comment by Vladislav4KZ** [comment]
default.cfg has an alias for zoom on the F11 key.

If GoldSource or Xash3D ever executed default.cfg, then a zoom alias should be bound to the F11 key.
